frontend/src/metabase/components/SegmentListLoader.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { SegmentApi } from "metabase/services";
+
+class SegmentListLoader extends React.Component {
```

Comment:
Any reason not to use `EntityListLoader` here (and other `*ListLoader`components ?)

Reply:
I should just `git rm` these for now. They were originally created for a later stage of the project that isn't currently accessible anywhere in the UI.